packages/app/styles/LogTable.module.scss

```diff
@@ -16,3 +16,17 @@
     font-weight: bold;
   }
 }
+
+.expandedRowContent {
```

Comment:
I'm not sure what's going on here but i think the lack of a fixed height is causing the table to jitter on scroll, likely due to rapid mounting/unmounting due to virtualization with the variable height is my guess

![Kapture 2025-09-05 at 23 42 26](https://github.com/user-attachments/assets/af6bc9b2-d94e-4778-b29e-6af87b7162fe)